specs/integration/config.js

```diff
@@ -0,0 +1,13 @@
+/* You can overwrite this config using the --config command-line option.
+See: https://github.com/MattiSG/Watai/wiki/Configuration
+E.g.: $ watai specs/integration/  --config '{"baseURL":"http://example.org"}'
+
+Note that you will have to set at least these config options
+in order to perform authenticated operations:
+
+$ --config '{"email":"david@larlet.fr","password":"testpassword","username":"David Larlet"}'
```

Comment:
Why not store this in the config directly, rather than as a comment? Furthermore, that would make tests runnable without a CLI switch.